src/main/java/org/folio/circulation/resources/RequestNoticeSender.java

```diff
@@ -57,20 +55,6 @@ public RequestNoticeSender(PatronNoticeService patronNoticeService) {
       .withNoticeContext(createRequestNoticeContext(request))
       .build();
     patronNoticeService.acceptNoticeEvent(requestCreatedEvent);
-
-    if (request.getRequestType() == RequestType.RECALL &&
```

Comment:
Rather than moving this, we could split the decision to send and the sending.

The sending could remain here and the check could be in a method in UpdateLoan.

What do you think?

Reply:
Agree. In case loan has  `hasDueDateChanged()` method, sending a notice can stay here

Reply:
As I mentioned in the thread above, saving info about the taken decision causes difficulties.  Also `UpdateLoan#onRequestCreation()` seems to me to be the place, where _Recall notice_ can be emitted because the causing event is due date change. What do you think?